plugins/vbe/README.md

```diff
@@ -93,7 +89,7 @@ VBE requires the board firmware to provide information about the firmware within
 the device tree passed to the Operating System.
 
 For systems without device tree, currently the only option is to install a file
-for use by fwupd, typically in `/var/local/lib/fwupd/vbe/system.dtb`.
+for use by fwupd, typically in `/var/lib/fwupd/vbe/system.dtb`.
```

Comment:
These changes seem to make it necessary to install fwupd / the plugin before it can be tested. Am I missing something?

Reply:
In general we don't advise users to install fwupd system-wide and instead encourage a prefix; in this case I thought it would be more obvious to show the distro-installed location rather than using something like `${PREFIX}/var/lib/fwupd/vbe/system.dtb` which might be even more confusing.

Reply:
Fair enough, but this info is for development, not users. Perhaps we should have a separate file?